examples/bevy_simple/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use bevy::prelude::*;
+use mun_runtime::Runtime as MunRuntime;
+
+const MUNLIB_PATH: &str = "mun/target/mod.munlib";
+
+fn main() {
+    App::new()
+        .add_plugins(MinimalPlugins)
+        .add_plugin(MunPlugin)
+        .run();
+}
+
+struct MunPlugin;
+
+impl Plugin for MunPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_startup_system(setup.exclusive_system())
+            .add_system(print_from_mun)
+            .add_system(reload_munlib_from_timer);
+    }
+}
+
+struct PrintTimer(Timer);
+struct MunReloadTimer(Timer);
+
+// Insert the mun runtime, and associated timers as world resources
+fn setup(world: &mut World) {
+    let builder = MunRuntime::builder(MUNLIB_PATH);
+    let mun: MunRuntime = unsafe { builder.finish() }.expect("Failed to spawn Runtime");
+    world.insert_non_send_resource(mun);
+    world.insert_resource(PrintTimer(Timer::from_seconds(1.0, true)));
+    world.insert_resource(MunReloadTimer(Timer::from_seconds(1.0, true)));
+}
+
+fn print_from_mun(mun: NonSendMut<MunRuntime>, time: Res<Time>, mut timer: ResMut<PrintTimer>) {
+    let result: usize = mun.invoke("mun_func", ()).unwrap();
+    if timer.0.tick(time.delta()).just_finished() {
+        println!("Printing value from `mun_func`: {}", result);
+    }
+}
+
+fn reload_munlib_from_timer(
+    time: Res<Time>,
+    mut timer: ResMut<MunReloadTimer>,
+    mut mun: NonSendMut<MunRuntime>,
+) {
+    if timer.0.tick(time.delta()).just_finished() {
```

Comment:
Im wondering, why did you add a timer here? The `update` function should be lightweight enough to call this on every tick. It internally already does optimizations that ensure this.

Reply:
- I included it because it is what I am using and it is proving to work well for me.
- There is not a performance or mun-related reason for choosing a timer-based update system.
- I can add a comment to the effect.

Reply:
Ah I see. The reason I asked is because I think that because this is an example people will likely copy it as a starting point. Since the timer serves no real purpose in the example this is code that just complicates the example. Therefore I would for the sake of showing people how to embed Mun in Bevy, remove it.

Reply:
Acknowledged. I will remove it.